src/types.ts

```diff
@@ -31,6 +31,12 @@ export interface Options {
    * @default 3070
    */
   port?: number
+  /**
+   * Whether to show semantic path when the file name is index.xxx .
+   *
+   * @default false
+   */
+  showSemanticPath?: boolean
```

Comment:
I think we can make this feature more universal. Not only file name includes `index`.

Provide an option named `extendedPathFileNames`.  It receives an array of strings. When the file name contains an element in the array, display expanded path. 
For example:

```js
{
   // @default []
    extendedPathFileNames: ['index', 'data']
}
```